test/jasmine/tests/toimage_test.js

```diff
@@ -179,7 +179,7 @@ describe('Plotly.toImage', function() {
         .then(function() { return Plotly.toImage(gd, {format: 'svg', imageDataOnly: true}); })
         .then(function(d) {
             expect(d.indexOf('data:image/')).toBe(-1);
-            expect(d.length).toBeWithin(32062, 1e3, 'svg image length');
+            expect(d.length).toBeWithin(36602, 1e3, 'svg image length');
```

Comment:
this length changed because of the additional font parameters?

Reply:
Good question.
Yes. But actually we could drop those that are set to `normal` from SVG to maintain the small size.